paddlets/datasets/splitter.py

```diff
@@ -306,11 +306,11 @@ class ExpandingWindowSplitter(SplitterBase):
         
     """
 
-    def __init__(self, n_splits: int = 5,
-                 test_size: Union[int, None] = None,
-                 skip_size: int = 0,
-                 max_train_size: int = None,
-                 verbose: bool = True) -> None:
+    def __init__(self, n_splits: Optional[int] = 5,
+                 test_size: Optional[Union[int, None]] = None,
+                 skip_size: Optional[int] = 0,
+                 max_train_size: Optional[int] = None,
+                 verbose: Optional[bool] = True) -> None:
```

Comment:
(1) Line309 n_splits should Not be Optional.
(2) Line310 can be either Optional[int] = None, or Union[int, None] = None. The reason is that Optional[x] is equivalent to Union[x, None].
(3) Line311 skip_size  should Not be Optional.
(4) Line313 verbose should Not be Optional.
Refers to https://docs.python.org/3/library/typing.html#typing.Optional